types/applications/baseEntitySheet.d.ts

```diff
@@ -79,15 +62,9 @@ declare namespace BaseEntitySheet {
   interface Data<O extends Entity = Entity> extends FormApplication.Data<O> {
     cssClass: string;
     editable: boolean;
-    entity: O extends Entity<infer D> ? D : never;
+    entity: O extends Entity<infer D> ? Duplicated<D> : never;
```

Comment:
It might be cleaner to just use `Duplicated<O['data']>`.

Reply:
done